### PR TITLE
chore: remove inactive sponsors, add Sentry

### PR DESCRIPTION
**Pull request by vytas7**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -87,23 +87,12 @@ <h1 class="block-title display-1">Falcon</h1>
...
-            <a href="https://www.salesforce.com" target="_blank"><img src="img/sponsors/salesforce.svg" height="30" alt="Salesforce"></a>
-        </p>
-
-        <p>
-            <a href="https://www.misaka.io" target="_blank" style="margin-right: 10px"><img src="img/sponsors/misaka.svg" height="30" alt="Misaka Network" style="filter: invert(1)"></a>
+            <a href="https://sentry.io/" target="_blank"><img src="img/sponsors/sentry.svg" height="60" alt="Sentry"></a>
 
-            <a href="https://github.com/LikaloLLC" target="_blank" style="margin-right: 10px"><img src="img/sponsors/likalo.png" height="30" alt="Likalo"></a>
+            <!-- Just adding a hidden logo so that it is served on HTTP. -->
+            <img src="img/sponsors/sentry-dark.svg" height="60" alt="Sentry (dark)" style="display: none;">
```
Comment (CaselIT) [inline src/index.html:95]: I don't follow what's the objective of this line

Comment (vytas7) [reply src/index.html:95]: I don't want this logotype to be displayed here, but I want it to be served on `/assets/sentry-dark.svg`, is there a better way?

Comment (CaselIT) [reply src/index.html:95]: don't all logo files get copied to the asset folder?

Comment (vytas7) [reply src/index.html:95]: No, only these that are referenced from the code it seems.

Comment (CaselIT) [reply src/index.html:95]: btw seems there's a way of changing the image depending on the theme: https://dev.to/ziratsu/switch-to-a-darker-image-when-on-dark-mode-2lkh

Comment (CaselIT) [reply src/index.html:95]: > No, only these that are referenced from the code it seems.

oh strange. well let's keep it like this

**Review by CaselIT — APPROVED** [review]

**Pull request merged**